log-viewer-frontend/src/app/log-navigator/log-navigator.component.ts

```diff
@@ -292,7 +292,10 @@ export class LogNavigatorComponent implements OnInit, AfterViewInit {
         } else {
             let fileType = fsItem.type;
 
-            if (fileType === 'log' || fileType === 'out' || fileType === 'text') {
+            if (fileType === 'log' || fileType === 'out' || fileType === 'text' ||
+                fileType === 'tgz' ||
+                fileType === 'unknown'  // tgz on linux detected as unknown!
```

Comment:
Why it's detected as unknown? `fileType === 'tgz'` if the file name matches the regexp in com.logviewer.files.FileTypes.TGZ.   The regexp is `.*\\.(tgz|tar\\.gz)` , what is the file name on linux?

Reply:
Files with `.gz` extension should not have `fileType === 'tgz'` , because they are not "rag gzip", they are "gzip".
I [added](https://github.com/sevdokimov/log-viewer/commit/6b602934abe59a0f05d183cce6f7a74f3502dd39) `FileTypes.GZ` file type for files with GZ extension. 

Do you want to hande "tar gzip" or "gzip" files? You cannot unpack "tar gzip" using `try (GZIPInputStream gis = new GZIPInputStream(Files.newInputStream(file.toFile().toPath()));`

Reply:
Yea you are correct.
It is only gz not tarred

I will check it

Reply:
Done